stackrox/base/services/kustomization.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

```suggestion
kind: Kustomization
namespace: stackrox
```
Move the namespace to the kustomization so it need only be modified in one place rather than every manifest, if we were to change it.